python/cugraph/cugraph/testing/mg_utils.py

```diff
@@ -35,6 +35,9 @@ def start_dask_client(
     device_memory_limit=0.8,
 ):
     dask_scheduler_file = os.environ.get("SCHEDULER_FILE")
+    dask_num_workers = os.getenv("DASK_NUM_WORKERS")
```

Comment:
Place holder for suggestion for swapping this out to `dask_worker_devices `

Reply:
Thanks, I changed to `dask_worker_devices` in commit [f4433f4](https://github.com/rapidsai/cugraph/pull/3540/commits/f4433f499403aa04e0393e9cef21306cbb5497ea)